processor/src/test/java/org/mapstruct/ap/test/dependency/GraphAnalyzerTest.java

```diff
@@ -0,0 +1,224 @@
+/**
+ *  Copyright 2012-2015 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.test.dependency;
```

Comment:
Ah, ok. What I grasped was that all those tests under ```test``` are more kind of integrative tests, not focusing on single classes. Whereas those tests that are not witin ```test``` are real unit tests that have one particular class under test - and those would usually be named ```fqn + "Test"```.
But those unit tests are quite rare for us, so such a separation is perhaps not really required for us.